dcgm/metadata.csv

```diff
@@ -0,0 +1,24 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+dcgm.dec_utilization,gauge,,,,Decoder utilization (in %).,0,dcgm,,
```

Comment:
There is some information still missing for all metrics:

1. unit
2. short description

Have a look at any existing integration's metadata to see examples of both.